siliconcompiler/tools/opensta/timing.py

```diff
@@ -25,9 +25,16 @@ def __init__(self):
                            defvalue="tools/_common/sdc/sc_constraints.sdc",
                            package="siliconcompiler")
 
+        self.add_parameter("device_type",
+                           "<asic,fpga>",
+                           "The type of device that the tool is targeting")
```

Comment:
I think instead of this (which is not really user settable anyways. you can overwrite:
`get_tcl_variables` and add a variable (`opensta_timing_mode` = <asic,fpga>)